step_definitions/topic-steps.js

```diff
@@ -19,24 +20,39 @@ When(/^.* clicks on the topic with name '([^']*)'$/, function (topicName) {
 Then(/^.* should see that the topic with name '([^']*)' is visible on the topic page$/, async function (topicName) {
 	return addEditTopicPage.isTopicTitleVisible(topicName);
 });
+
 When(/^.* clicks on the pencil button in the line of the topic with name '([^']*)' to edit the topic$/, async function (topicName) {
 	await addEditTopicPage.clickOnTopicEditPencilButton(topicName);
 });
+
 When(/^.* changes topic name '([^']*)'$/, async function (changedTopicName) {
 	await addEditTopicPage.setTopic(changedTopicName);
 });
+
 When(/^.* finds title '([^']*)' and changes title on '([^']*)' and text '([^']*)' of the topic$/, async function (contentTitle, changeContentTitle, changeDescription) {
 	await addEditTopicPage.setContentByTitle(contentTitle, changeContentTitle);
 	await addEditTopicPage.setDescription(changeDescription);
 	await addEditTopicPage.clickCreateTopicButton();
 });
+
+When(/^.* clicks on the trashcan button in the line of the topic with name '([^']*)' to delete the topic$/, async function (topicName) {
```

Comment:
in my opinion it is too long. My suggestion is:
clicks trashcan in line of topic named '([^']*)'

Reply:
changed